### PR TITLE
Adds Password Reset Redirect

### PR DESCRIPTION
**Pull request by moebrowne**
This PR adds a redirect from `/.well-known/change-password` to `/auth/reset` to support the new [spec from W3C](https://w3c.github.io/webappsec-change-password-url/)

Related: https://web.dev/change-password-url/

### CODE REVIEW

**Comment by mikemand** [comment]
Hi @moebrowne,

You can use `Route::permanentRedirect` (or rather `$router->permanentRedirect`) here, instead of making a new controller method: https://laravel.com/docs/5.7/routing#redirect-routes

**Comment by mikemand** [comment]
Looks good. 👍 What made you decide to use `redirect` instead of `permanentRedirect`? I am not super aware of the nuances between 301 and 302 redirects, so I don't know which one would be better here.

**Comment by moebrowne** [comment]
In this case the W3C spec specifically calls for a temporary redirect, presumably because the change password URL could change in the future.

A 301 redirect is permanent and will be cached by browsers and a 302 redirect is temporary.

**Comment by mikemand** [comment]
Ah, I see. Thanks for clearing that up. 😃

@nWidart This looks good to me. 🚀

File: Modules/User/Http/frontendRoutes.php [hunk 1]
```diff
@@ -22,3 +22,5 @@
     # Logout
     $router->get('logout', ['as' => 'logout', 'uses' => 'AuthController@getLogout']);
 });
+
+$router->redirect('.well-known/change-password', route('reset'));
```
Comment (nWidart) [inline Modules/User/Http/frontendRoutes.php:26]: looks like there's a new line missing here at EOF.

**Comment by moebrowne** [comment]
@nWidart I have added the additional new line and fixed the merge conflicts

**Comment by nWidart** [comment]
Perfect, thanks!

**Pull request merged**